ocaml/typing/printtyp.mli

```diff
@@ -181,6 +181,8 @@ val print_items: (Env.t -> signature_item -> 'a option) ->
    for Foo__bar. This pattern is used by the stdlib. *)
 val rewrite_double_underscore_paths: Env.t -> Path.t -> Path.t
 
+val rewrite_double_underscore_longidents: Env.t -> Longident.t -> Longident.t
+
```

Comment:
Why is this function needed?

Reply:
Because `transl_extension_constructor` (among many other things) now takes a `Longident.t` rather than a `Path.t` and it needs to do the same rewrite it did before.